pgdb/v1/fts.go

```diff
@@ -136,14 +115,47 @@ func camelSplitDocs(docs []*SearchContent) []lexeme {
 			continue
 		}
 		docValue := interfaceToValue(doc.Value)
-		for _, match := range camelCaseRegex.FindAllStringIndex(docValue, -1) {
-			result := docValue[match[0]:match[1]]
-			rv = append(rv, lexeme{strings.ToLower(result), match[0] + 1, doc.Weight})
+		var word []rune
+		for i, r := range docValue {
```

Comment:
can we make this a small function that we can directly unit test?

Reply:
@pquerna tests and fixes for things tests found: in 93e4ee3ab1cca0577c7e72b3efa1ebd4c01b81f6

Reply:
Added even more tests